modules/charon-core/src/main/java/org/wso2/charon3/core/protocol/endpoints/ResourceManager.java

```diff
@@ -43,8 +43,7 @@
      * client and server views of the new Resource. When a Resource is created, its uri must be returned
      * in the response Location header.}
      */
-    public SCIMResponse create(String scimObjectString, UserManager userManager,
-                               String attributes, String excludeAttributes);
+    SCIMResponse create(String scimObjectString, UserManager userManager, String attributes, String excludeAttributes);
```

Comment:
Exceeds line length.

Reply:
It seems like, but in the editor it does not exceed.

Reply:
It seems like here it's exceeding. But in the editor it does not exceed.